### DISCUSSION TITLE
How do i submit assignments?

### DISCUSSION THREAD

**Discussion by smngvlkz** [microsoft/Web-Dev-For-Beginners#1293] [General]
Hey people, kindly assist. I am new here. How do I go about submitting an assignment?

**Comment by Hashira30** [comment]
Welcome, I’d like to help but I’m also a noob. Lmao

**Reply by smngvlkz** [reply]
@Stormbreaker111 Why man, why? I received the notification and rushed thinking I was about to get assistance😂😂
